.vscode/settings.json

```diff
@@ -0,0 +1,76 @@
+{
+    "files.associations": {
```

Comment:
Can you remove .vscode folder?

Reply:
Yes. Thanks for pointing that out.

Reply:
@anonrig To be fair, it should be in our .gitignore.

Reply:
I agree, but should we add all other IDE configuration folders as well? Such as .idea?